tests/userTests.py

```diff
@@ -322,4 +322,22 @@ def test_update_user(self):
         moreBadInput = {"status": "approved", "is_active": False, "permissions": "agency_admin"}
         response = self.app.post_json("/v1/update_user/", moreBadInput, expect_errors=True,
                                       headers={"x-session-id": self.session_id})
-        self.check_response(response, StatusCode.CLIENT_ERROR)
\ No newline at end of file
+        self.check_response(response, StatusCode.CLIENT_ERROR)
+
+    def test_email_users(self):
+        """ Test email users """
+        self.login_approved_user()
+        input = {"users": [self.agency_user_id], "submission_id": self.submission_id,
+                 "email_template": "review_submission"}
+        response = self.app.post_json("/v1/email_users/", input, headers={"x-session-id": self.session_id})
+        self.check_response(response, StatusCode.OK, "Emails successfully sent")
+
+        badInput = {"users": [self.agency_user_id]}
+        response = self.app.post_json("/v1/email_users/", badInput, expect_errors=True,
+                                      headers={"x-session-id": self.session_id})
+        self.check_response(response, StatusCode.CLIENT_ERROR)
+
+        badInput = {"users": [self.agency_user_id], "submission_id": -1}
```

Comment:
Since we tested for an invalid submission_id, we should also test for invalid users and invalid email templates too